is_core/generic_views/form_views.py

```diff
@@ -571,13 +573,35 @@ def get_success_url(self, obj):
         else:
             return self.request.get_full_path()
 
+    def _get_detail_export_types(self):
```

Comment:
pridavam sem media_type - kterej je treba mit v url, v tej puvodni verzi se to nejak dela pres js jeste na tom onclicku - to jsem jednak nebyl schopnej pobrat jak se to tam dela a hlavne je to podle mne zbytecny

Reply:
Tam to dělal JS, je to tam trochu komplikované s tím souhlasím

Reply:
rozděl to na getter a generator 

(get_detail_export_types a generate_detail_export_types) ten generátor tam bude dávat už ty content typy z pystonu.

Reply:
pak můžeš mít rozdíly v názvu detail_export_types a generated_export_types

Reply:
navíc podle mě je slovo detail zbytečné, když view slouží pro detail objektu ne? Slovíčko detail bych nechal jen v metodě na core